### ISSUE TITLE
PEP 541 Request: oculus

### ISSUE THREAD

**Issue by ppfeister** [pypi/support#4390]
### Project to be claimed

`oculus`: https://pypi.org/project/oculus

### Your PyPI username

`ppfeister`: https://pypi.org/user/ppfeister

### Reasons for the request

Account was created at the end of Jun 23, 2022, and it disappeared on Sep 18, 2022. During this time, the account holder created several near-empty projects with short names. These projects all appear near-empty.

The 'Oculus' project in question here has a grand total of 18 lines of code (including the setup.py and documentation). The only true piece of code is something to the effect of a `x = y**z`. There is no SCM, Readme, contact information, links, etc, tied to the project or package. When querying the package downloads, it only has the occasional hit from bots. It's only release was a version 0.0.1 in June of 2022, similar to every other project this account has opened (excluding the two "newest" posted a month later, which are random uploads of unrelated projects).

The entire account is abandoned and void of any readily apparent contact information.

__Edit:__ After reviewing more packages, it seems that every package bar one(?) on the account is likely to be a duplicate of the same 18 lines, barring the different name.

### Maintenance or replacement?

Replacement

### Source code repositories URLs

Original project does not use any SCM. See "Reasons for the request" above for more info.

Replacement project can be found at https://github.com/ppfeister/oculus (devel)

### Contact and additional research

No successful contact has been made with the owner.
\- Account has no available contact information
\- Account uses a generic username, rather than highly unique and easily traceable
\- Account has several projects from the same time period, none of which seem to have any contact information on the project page
\- No contact information found in the tarballs for the packages I checked

__Edit:__ I have been able to find an old reddit account ([u/redditfan45655](https://www.reddit.com/user/redditfan45655/)), but that account is 3 years unused. Some posts on the account indicate that he was (at the time of posting) only 15. There are several other discord usernames that he's discussed in comments on that account, seemingly to circumvent bans on several Minecraft-related discord servers. All of these mentioned accounts seem to be closed and unreachable. Researching each of the discovered usernames returns no valuable results. This is the closest we've come, and there's still no way to actually reach this person.

__Edit 2:__ Through some careful sleuthing, I've discovered this now-private youtube [video](https://www.youtube.com/watch?v=yn7YGN7nF5A), [channel](https://www.youtube.com/@inteldev), and a [Hypixel](https://hypixel.net/members/__intel__.3831124/) account. Still without any means of contact -- just dead accounts with no info.

Account was only active for a month or two, creating all these projects without content, before abandonment.

### Code of Conduct

- [X] I agree to follow the PSF Code of Conduct

**Comment by ppfeister** [comment]
Hey @cmaureir -

It doesn't seem like many of the old guys are around anymore. Curious -- have you become the only real active maintainer in recent months? Surprised to see fewer active contributors compared to this time a year or two ago

**Comment by ppfeister** [comment]
Hey @di -

Noticed that you've returned to support recently. I think that this would be a pretty cut and dry resolution due to the clear pattern of mass squatting (#4391) and complete lack of content. It currently serves as a blocker for me on other projects.

Seems that y'all are a little short handed nowadays, so is there anything that I could do on my end to help process this req? I'm continuing to search for means of contact, and have actually recruited a couple other people in that effort as well.

**Comment by Thespi-Brain** [comment]
Hi @ppfeister,

We are working to contact the owner to determine if they're reachable and have sent them a courtesy message.

Final determination regarding project ownership will be made by the PyPI Administrators when the research process is complete.

**Comment by Thespi-Brain** [comment]
**oculus** has been deemed invalid. Additionally, the owner did not respond back regarding the status of their project. 

I recommend that the PyPI Admins release the name for future use.

**Comment by Thespi-Brain** [comment]
The package owner wrote back stating they "added functionality" although, comparing the first and second versions of the package, it doesn't seem like that big of a change. Also reporting this package owner as a mass name squatter. Up to the discretion of PyPI admins on how they wish to proceed.

**Comment by ewdurbin** [comment]
I agree that this is still name squatting. Added 'ppfeister' as 'Owner' on 'oculus'
